### PR TITLE
PRIME-1608 nursing type groupings

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

File: prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html [hunk 1]
```diff
@@ -35,7 +35,23 @@
                       [ngTemplateOutlet]="collegeLicences"></ng-container>
       </div>
 
-      <ng-container *ngIf="collegeCode.value">
+      <div class="col-sm-12">
+        <ng-container *ngIf="!condensed && collegeCode.value && collegeCode.value === CollegeLicenceClassEnum.BCCNM">
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html:39]: Looks like you can drop one of the `college.value` and instead have `collegeCode?.value === CollegeLicenceClassEnum.BCCNM`

Comment (hogmann) [reply prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html:39]: very true, probably had my brain in a  knot from all the conditionals i was doing.

**Review by mtpultz — APPROVED** [review]

File: prime-dotnet-webapi/Models/Certifications/CollegeLicenseGrouping.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+    [Table("CollegeLicenseGrouping")]
+    public class CollegeLicenseGrouping
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/Certifications/CollegeLicenseGrouping.cs:7]: should probably implement ILookup<int>
Also, table name should probably be CollegeLicenseGrouplingLookup to be consistent with our naming convention

File: prime-dotnet-webapi/Models/Certifications/CollegeLicense.cs [hunk 3]
```diff
@@ -13,5 +13,8 @@ public class CollegeLicense
 
         [JsonIgnore]
         public License License { get; set; }
+        public int? CollegeLicenseGroupingCode { get; set; }
+        [JsonIgnore]
+        public CollegeLicenseGrouping CollegeLicenseGrouping { get; set; }
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/Certifications/CollegeLicense.cs:18]: These lines should have newlines between them, to keep the spacing consistent.

**Review by james-hollinger — APPROVED** [review]
L Grouped TM

**Pull request merged**